src/ngraph/builder/quantization_util.hpp

```diff
@@ -99,6 +99,55 @@ namespace ngraph
                     (std::pow(2, -24) * static_cast<double>(max_abs32 / max_abs8)));
                 return scale;
             }
+
+            template <typename T>
+            static inline T get_quantization_scale(const std::shared_ptr<Node> min_input,
+                                                   const std::shared_ptr<Node> max_input,
+                                                   const ngraph::element::Type& type,
+                                                   bool is_quantize = false)
+            {
+                auto min_input_const_op =
+                    std::dynamic_pointer_cast<ngraph::op::Constant>(min_input);
+                auto max_input_const_op =
+                    std::dynamic_pointer_cast<ngraph::op::Constant>(max_input);
+
+                if (min_input_const_op == nullptr)
+                    throw ngraph_error("min_input shouldn't be nullptr!");
+                else if (max_input_const_op == nullptr)
+                    throw ngraph_error("max_input shouldn't be nullptr!");
+
+                auto input_min_range = min_input_const_op->get_vector<T>();
+                auto input_max_range = max_input_const_op->get_vector<T>();
+
+                T min_range;
+                T max_range;
+                if (is_quantize)
```

Comment:
There may come a time, some day, where we want to adjust min/max for Dequantize as well.  Hence, this flag could take a more generic name based on what this code is doing:  adjusting a too-close min/max by delta.